src/components/Input/Input.vue

```diff
@@ -192,11 +206,6 @@ export default defineComponent({
       this.hovered = false
       this.active = false
     },
-    handleKeydown(e: Event): void {
-      if (this.disabled) return
-      this.validate(e)
-      e.target?.blur()
```

Comment:
do we want to keep the blur functionality for enter?

Reply:
Not necessarily - for example I'm attempting to use this to input a series of tags, where after each tag is complete the user is expected to press enter and then continue typing. As written, this would require that they need to re-focus the input after each one; I'd rather the consumer handle blurring the input on enter.

Reply:
ah good counter 👍